rhizome/host/lib/vm_setup.rb

```diff
@@ -83,8 +84,35 @@ def setup_networking(skip_persisted, gua, ip4, local_ip4, nics, ndp_needed, mult
     forwarding
   end
 
+  def unblock_ip4(ip4)
+    ip_net = NetAddr::IPv4Net.parse(ip4).network.to_s
+    filename = "/etc/nftables.d/#{q_vm}.conf"
+    temp_filename = "#{filename}.tmp"
+    File.open(temp_filename, File::RDWR | File::CREAT) do |f|
+      f.flock(File::LOCK_EX)
```

Comment:
I don't think here it would have the same issue. This is running on the dataplane side and a single threaded program. Each file is also specific to the vm.